man/coap-server.txt.in

```diff
@@ -39,6 +40,18 @@ OPTIONS
    Use the specified PEM file which contains the CERTIFICATE and PRIVATE
    KEY information. This argument requires (D)TLS with PKI to be available.
 
+*-C* cafile::
+  PEM file containing the CA Certificate that was used to sign the certfile.
+  If defined, then the client will be given this CA Certificate during the
+  TLS set up.  Furthermore, this will trigger the validation of the client
+  certificate.  If certfile is self-signed, then also add '-C certfile' on the
```

Comment:
Actually original text was correct in that you need '-c certfile -C certfile' for a self-signed certificate to be able to do certificate verification. However, text updated for clarity to
````
*-C* cafile::
  PEM file containing the CA Certificate that was used to sign the certfile
  defined using '-c certfile'.
  If defined, then the client will be given this CA Certificate during the
  TLS set up.  Furthermore, this will also trigger the validation of the client
  provided certificate against this cafile or using the CAs defined by
  '-R root_cafile'.
  If certfile is self-signed, then also add '-C certfile' (as this cafile
  'signed' the certfile) on the command line to trigger validation.
````

Reply:
Same problem as above for self-signed.

Reply:
This also looks good.